server/odc-service/src/main/java/com/oceanbase/odc/service/db/DBTableService.java

```diff
@@ -183,6 +185,19 @@ public GenerateTableDDLResp generateUpdateDDL(@NotNull ConnectionSession session
                 .build();
     }
 
+    public SuccessResponse syncExternalTableFiles(@NotNull ConnectionSession connectionSession, String schemaName,
+            @NotBlank String externalTableName) {
+        DBSchemaAccessor schemaAccessor = DBSchemaAccessors.create(connectionSession);
```

Comment:
may not use accessor directly in service layer, use extension instead please

Reply:
ok